sdk/storage/azure-storage-blob/azure/storage/blob/_serialize.py

```diff
@@ -57,6 +57,7 @@
     '2023-05-03',
     '2023-08-03',
     '2023-11-03',
+    '2024-05-04',
```

Comment:
We should re-generate Blob, Data, and Queue from the Swagger file in main just to pick up latest generator changes.